### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by CharlieArlott** [community/community#74328] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I had been trying to activate the copilot extension in the VS Code and codespaces, but I still keep getting the "Invalid copilot token: missing token: 403" output.

In case you wonder, I had tried the following troubleshooting methods:

1. Signing in and out of my github account on VS Code and browser.
2. Making sure that my github account has access to the Github Copilot (My account has access to the Student Developer Pack)
3. Emptying the cache of my browser.
4. Uninstalling and Installing the VS Code.
5. Restarting my computer.
6. Uninstalling and Installing the Copilot extension.

The following attached picture is the screenshot of the output log:

![Screenshot 2023-11-07 at 7 54 31 PM](https://github.com/community/community/assets/81029098/83eec417-f6c6-4526-b2d9-940d0827624c)

I really hope that anyone can help me to resolve this issue.

Thank you.

**Comment by davecheney** [comment, marked as answer]
Please visit https://github.com/settings/copilot and complete the required setup.

**Comment by Ryanyamg** [comment]
Have you solved the problem

**Reply by CharlieArlott** [reply]
Yes, I have.

**Reply by Ryanyamg** [reply]
I have the same problem. How did you solve it？

**Reply by Ryanyamg** [reply]
please,thanks！！

**Reply by CharlieArlott** [reply]
![Screenshot 2024-03-14 at 8 15 55 PM](https://github.com/community/community/assets/81029098/7710ccd0-3eb6-4fd0-941d-cf9555e8e966)
Go to this link of yours: https://github.com/settings/copilot
Try changing that part to 'allowed'.

**Reply by Ryanyamg** [reply]
alway allow

**Reply by CharlieArlott** [reply]
can you show me your screenshot?

**Reply by Ryanyamg** [reply]
wait a second

**Reply by Ryanyamg** [reply]
![image](https://github.com/community/community/assets/150518568/becdbf95-52b0-4559-bcd9-116996d29b15)

**Reply by CharlieArlott** [reply]
Is your account github pro?

**Reply by Ryanyamg** [reply]
i do not down   is it
?

**Reply by CharlieArlott** [reply]
By looking at the top part of your screenshot, it seems like your account is not github pro.

**Reply by Ryanyamg** [reply]
I used to be able to use it, but suddenly not today, I don't know what the problem

**Reply by alalshow** [reply]
저도 같은 문제인데 문제해결하신분있나요?

**Reply by bubumua** [reply]
have you solved the problem? I met the same problem, just suddenly unable to use yestoday.
